src/win/fs-event.c

```diff
@@ -225,11 +225,11 @@ int uv_fs_event_start(uv_fs_event_t* handle,
   }
 
   if (!handle->buffer) {
-    handle->buffer = (char*)_aligned_malloc(uv_directory_watcher_buffer_size,
-                                            sizeof(DWORD));
```

Comment:
Hm, now that I look at this, I am wondering why libuv is using `_aligned_malloc` at all. This appears to be the only place in the code where `_aligned_malloc` is used.

The [MSVC compiler](https://msdn.microsoft.com/en-us/library/6ewkz86d.aspx) is guaranteed to align on 8 byte boundaries for 32-bit code and 16 byte boundaries for 64-bit code. So 4-byte alignment is not needed here.

I'm having a harder time proving the same is true for mingw, but everything I'm googling so far seems to suggest the same alignment patterns for malloc in mingw.

It would be nice if we could avoid the _aligned APIs altogether; then I could remove the custom version.